1228-public_page/user_navigates_to_landing_page_for_registration_and_can_see_files.feature

```diff
@@ -0,0 +1,51 @@
+Feature: User navigates to Landing Page for Registration and can see files
+
+    @1530
+    Scenario: Files that are Administrative Agreements are hidden
+        Given a User views the Landing Page for a Registration
+        And the Registration contains a File, which is an Administrative Agreement
+        When they view the Files section
+        Then they do not see the File that is an Administrative Agreement
+
+    Scenario: Files that are part of Registration are listed
+        Given a User views the Landing Page for a Registration
+        And the Registration contains Files
+        When they view the Files section
+        Then they can see files that are not Administrative Agreement are listed
+        And for each file they can see:
+            | name    |
+            | size    |
+            | version |
+            | license |
+        And they can download files that are not embargoed
+
+    @2158
+    Scenario: Files can be previewed
+        Given a User views the Landing Page for a Registration
+        And the Registration contains Files that are not embargoed
+        And every file has an expandable Preview panel
+        When the user expands the Preview panel
+        Then the selected file is downloaded
+        And the downloaded file is of type "<FileType>"
+        And the downloaded file is displayed
+        Examples:
+            | FileType         |
+            | PDF              |
+            | Image            |
+            | Microsoft Office |
```

Comment:
Er `Microsoft Office` en filtype?

Reply:
Ikke akkurat... Men vi skal kunne vise Office-filer, og det er jo ganske mange typer (også typer fra Open Office, Libre Office, etc): https://docs.microsoft.com/en-us/deployoffice/compat/office-file-format-reference
noe bedre forslag til samlebetegnelse?

Reply:
skal vi vise _alle_ MS-filer? jeg ville foretrukket en opplisting av hvilke vi skal støtte

Reply:
`Office Files`?

Reply:
Opplisting av hva vi støtter blir jo i praksis de som står i linken da, så det blir jo en del :p 
samme gjelder forsåvidt Image, som da blir JPG, PNG, SVG, GIF, etc. jeg syns egentlig det er like greit å være litt for raus i starten så blir det en positiv overraskelse om vi kan vise forhåndsvisning av litt sære typer, heller enn at vi må inn å oppdatere scenario og endre kode for å støtte en ny filtype som egentlig allerede kunne vært støttet

Reply:
Til info må ikke vi gjøre noen jobb for å støtte MS-filer, vi viser bare iframe med MS Web Viewer, eksempel: https://view.officeapps.live.com/op/embed.aspx?src=https://calibre-ebook.com/downloads/demos/demo.docx

så hva som inngår der fikser den selv, men vi må finne hvilke filer som skal sendes til den viseren, feks filer med mime type som inneholder "officedocument"

Reply:
oki da